konfera/urls.py

```diff
@@ -19,4 +19,5 @@
 urlpatterns = [
     url(r'^events/', include('konfera.event.urls'), name='event_list'),
     url(r'^register/', include('konfera.register.urls')),
+    url(r'^meetups/', 'views.meetup_list', name='list_all_meetups'),
```

Comment:
Thanks for the feedback! I'll definitely go back and update - I saw the Travis tests had failed. Once I am able to run the app I'll be able to fix and test, just wanted to get an early PR out there.